### ISSUE TITLE
Write up guide for organizers to reference when creating new posts for the Power Report

### ISSUE THREAD

**Issue by marcondag** [debtcollective/parent#154]
Title says it all. Would be nice to have some step-by-step pictures if anything is specific. Otherwise, just a list of the metadata tags for each category and what they do would be sufficient. Siteleaf takes care of making it easy to create a post in any category anyway. It's just the metadata that could be confusing.

Questions like what `youtube-id` and `interviewee-picture` should be need to be answered. Same goes for the metadata for the artwork category.

Marking as low priority because it doesn't need to be done before launch but should be done soon after.

**Comment by marcondag** [comment]
Done here: https://github.com/debtcollective/powerreport.debtcollective.org/wiki
